src/constants/entitiesConfig/EmptyEntitiesListContent/EmptyTestsListContent.tsx

```diff
@@ -8,6 +8,8 @@ const EmptyTestsListContent: React.FC<{action: () => void}> = props => {
       title="Create your first test in a few easy steps."
       description="Simply define your test, add any variables, execute it and view the results!"
       buttonText="Add a new test"
+      emptyListReadonlyTitle="No tests found"
+      emptyListReadonlyDescription="We could not find any tests in this environment."
```

Comment:
Or maybe this text not supposed to be showed and here just for code equality?

Reply:
It should not happen on OSS, so it's mostly to align the code and ensure that it will not fail.

I believe that we could think of changing the texts when there will be Cloud and OSS merged, will take note on that 👍